tests/php/PDepend/Input/ExcludePathFilterTest.php

```diff
@@ -219,4 +298,39 @@ protected function createFilteredFileList(array $excludes): array
 
         return $actual;
     }
+
+    /**
+     * Returns a random string with a given length.
+     *
+     * @param int $length The length of the random string.
+     *
+     * @return string
+     * @throws \Random\RandomException
```

Comment:
`RandomException` was introduced in PHP 8.2, so for comparability:
```suggestion
     * @throws Exception
```